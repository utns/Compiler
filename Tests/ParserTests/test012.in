var
    a: integer;

begin
    a := 1;
    if a then
        a := -1;
end.