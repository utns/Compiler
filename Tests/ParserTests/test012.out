begin
____1
:=
____a
if
____a
then
          ____1
     ____-
___:=
     ____a
end
